datafusion/optimizer/src/replace_distinct_aggregate.rs

```diff
@@ -54,8 +54,6 @@ use datafusion_expr::{Aggregate, Distinct, DistinctOn, Expr, LogicalPlan};
 /// )
 /// ORDER BY a DESC
 /// ```
-
-/// Optimizer that replaces logical [[Distinct]] with a logical [[Aggregate]]
```

Comment:
do we need the comment to be removed?

Reply:
it duplicates the first line of the long comment above, i don't think it's needed